src/widgets/TreeGrid.js

```diff
@@ -1,118 +1,131 @@
+import _ from 'lodash';
 import AccessibilityModule from '@curriculumassociates/createjs-accessibility';
 import TreeGridRow from './TreeGridRow';
 
 const OFFSET = 20;
 
 export default class TreeGrid extends createjs.Container {
   constructor(data, tabIndex) {
-    super(data, tabIndex);
+    super();
+    _.bindAll(this, '_onCollapseRow', '_onExpandRow');
+
     AccessibilityModule.register({
       displayObject: this,
+      role: AccessibilityModule.ROLES.TREEGRID,
+      events: [
+        {
+          eventName: 'collapseRow',
+          listener: this._onCollapseRow,
+        },
+        {
+          eventName: 'expandRow',
+          listener: this._onExpandRow,
+        },
+      ],
+    });
+
+    this._table = new createjs.Container();
+    AccessibilityModule.register({
+      displayObject: this._table,
       role: AccessibilityModule.ROLES.TABLEBODY,
     });
-    this.tabIndex = tabIndex;
-    this.data = data.rows;
```

Comment:
Some of this diff is just renaming things to start with `_` to designate what should be private members from public ones.